db/re/item_db.conf

```diff
@@ -61939,6 +61940,11 @@ item_db: (
 	Slots: 1
 	Loc: 256
 	View: 525
+	Script: <"
+		bonus bDex, 2;
+		skill TF_DOUBLE, 5;
```

Comment:
When description says "activate it at that level" it doesn't refer to enable skill to a higher level, but to the chance to do a double attack, and that refer to bonus bDoubleRate, wich uses the maximum skill level learned for that. It's like when having two buffs of diferent level but the same kind, only is applied the higher one.